docs/Basic-Guides.md

```diff
@@ -0,0 +1,142 @@
+
+![Set Brain to External](images/mlagents-SetExternalBrain.png)
+
+Next, we want the set up scene to play correctly when the training process 
+launches our environment executable. This means:
+* The environment application runs in the background
+* No dialogs require interaction
+* The correct scene loads automatically
+ 
+1. Open Player Settings (menu: **Edit** > **Project Settings** > **Player**).
+2. Under **Resolution and Presentation**:
+    - Ensure that **Run in Background** is Checked.
+    - Ensure that **Display Resolution Dialog** is set to Disabled.
+3. Open the Build Settings window (menu:**File** > **Build Settings**).
+4. Choose your target platform.
+    - (optional) Select “Development Build” to
+    [log debug messages](https://docs.unity3d.com/Manual/LogFiles.html).
+5. If any scenes are shown in the **Scenes in Build** list, make sure that 
+the 3DBall Scene is the only one checked. (If the list is empty, than only the 
+current scene is included in the build).
+6. Click *Build*:
+    a. In the File dialog, navigate to the `python` folder in your ML-Agents 
+    directory.
+    b. Assign a file name and click **Save**.
+
+![Build Window](images/mlagents-BuildWindow.png)
+
+Now that we have a Unity executable containing the simulation environment, we 
+can perform the training. You can ensure that your environment and the Python 
+API work as expected, by using the `python/Basics` 
+[Jupyter notebook](Background-Jupyter.md) introduced in the next section.
+
+## Using the Basics Jupyter Notebook
+
+The `python/Basics` [Jupyter notebook](Background-Jupyter.md) contains a 
+simple walkthrough of the functionality of the Python 
+API. It can also serve as a simple test that your environment is configured
+correctly. Within `Basics`, be sure to set `env_name` to the name of the 
+Unity executable you built earlier.
+
+More information and documentation is provided in the 
+[Python API](Python-API.md) page.
+
+## Training the Brain with Reinforcement Learning
+
+Go to your command line, enter the `ml-agents/python` directory and type: 
+
+```
+python3 learn.py <env_name> --run-id=<run-identifier> --train 
+```
+
+![Training command example](images/training-command-example.png)
+
+**Note**: If you're using Anaconda, don't forget to activate the ml-agents environment first.
+
+The `--train` flag tells ML-Agents to run in training mode. `env_name` should be the path to the Unity executable that was just created. 
+
+If the learn.py runs correctly and starts training, you should see something like this:
+
+![Training running](images/training-running.png)
+
+You can press Ctrl+C to stop the training at anytime, and your trained model will be at `ml-agents/python/models/<run-identifier>/<env_name>_<run-identifier>.bytes`. You can now embed this trained model into your internal brain by following the steps below, which is similar to the steps described [above](#play-an-example-environment-using-pretrained-model). 
```

Comment:
Well, not quite anytime. Don't you have to wait for it to reach a checkpoint.

Reply:
Maybe something like:

"You can press Ctrl+C to stop the training at anytime, and if the training has went through a checkpoint, you can find your trained model will be at ``..."

?